### PR TITLE
Add support for overriding base mac address, and more

### PR DESCRIPTION
**Pull request by troglobit**
This PR is a bit of a mix between cleanup and adding functionality.  The latter is covered in the below description and the former I hope stand for themselves.

 1. Adjust calculated MAC address offsets to favor Infix/Infamy DUT names.  I.e., dut1 port e1 will now get 02:00:00:00:01:01 and dut2 port e7 02:00:00:00:02:07
 2. Generate a mactab file for Infix so the interface names match the topology port names
 3. Add support for overriding the default mac oui, or a node's base mac, or a port mac

The README has been updated to reflect the new settings for mac address while dropping the Infix Classic from the tutorial.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,50 +46,68 @@ Tutorial
...
-
-        server:eth0 -- client1:eth0;
-        server:eth1 -- client2:eth0;
+    node [shape=record];
+    qn_template="infix-x86_64";
+    qn_append="quiet";
+    qn_mem="512M";
+    qn_oui="00:a0:85";
+
+    server [label="server   | { <e1> e1 | <e2> e2 }", qn_basemac="00:07:7c:00:00:00"];
```
Comment (wkz) [inline README.md]:
```suggestion
    server [label="server   | { <e1> e1 | <e2> e2 }", qn_basemac="02:de:ad:00:00:00"];
```

File: README.md [hunk 2]
```diff
@@ -46,50 +46,68 @@ Tutorial
...
+    client1 [label="client1 | { <eth1> eth1 }"];
+    client2 [label="client2 | { <eth1> eth1 }"];
+
+    server:eth0 -- client1:eth0 [qn_tailmac="00:07:7c:12:34:56", qn_headmac="08:00:20:c0:ff:ee"];
```
Comment (wkz) [inline README.md]:
```suggestion
    server:eth0 -- client1:eth0 [qn_tailmac="02:de:ad:12:34:56", qn_headmac="08:00:20:c0:ff:ee"];
```

**Review by wkz — COMMENTED** [review]
Thanks a lot!

I'll just replace one randomly chosen OUI for another equally random one :wink:

**Comment by troglobit** [comment]
No thank you, for taking the time to help out today and review!

I'll go ahead now and update the KernelKit clone for updating Infix as well. Cheers!

**Pull request merged**